lib/realtime/application.ex

```diff
@@ -42,6 +42,11 @@ defmodule Realtime.Application do
       name: Realtime.Registry.Unique
     )
 
+    Registry.start_link(
+      keys: :unique,
+      name: Realtime.Registry.Tenant
```

Comment:
I don't think we need a separate local process registry for this because syn is our process registry.

Reply:
you are right, I think this are leftovers of other experiments, deleting